app/models/user_unit.rb

```diff
@@ -1,2 +1,4 @@
 class UserUnit < ActiveRecord::Base
+	belongs_to :unit 
```

Comment:
take a look at your indention settings in sublime - make it two spaces so we don't get merge conflicts! :+1: